aries/aries_vcx/src/errors/mapping_indyvdr.rs

```diff
@@ -0,0 +1,49 @@
+use indy_vdr::{
```

Comment:
It seems that this is reason why `indy_vdr` was introduced as dependency - but `aries_vcx` shouldn't know anything about the underlying ledger implementation.

Reply:
Doesn't the mapping for `VcxLedgerError` in the file below suffice?

Reply:
This file is the reason why `indy_vdr` is now a dependency - it was moved from `aries_vcx_core`. There used to be `VdrError` -> `AriesVcxCoreErrr` -> `AriesVcxError` conversion path but because `aries_vcx_core` is going away, we need to change to a direct `VdrError` -> `AriesVcxError` mapping. Alternatively, we could create an error variant in `VcxLedgerError` for each `VdrError`.

Reply:
The mapping from `VcxLedgerError` currently does not suffice, because `VcxLedgerError` does not create its own variant for each `VdrError` but rather [wrapps it](https://github.com/hyperledger/aries-vcx/pull/1172/files#diff-e8ad81d5df90cb2b04e8295db885f1a06795ce78ee034fd6200824384bbb53a7R19)

Reply:
Lets do the changes necessary to shield off vcx from `indy-vdr`, be it creating/adjusting `VcxLedgerError`. There's not much code relying on specific ledger error kinds (afaik the only place where we directly deal with them is when we are creating credential definition, and we have some specific behaviour based on whether the same cred-definition already exists/not exist on ledger.

Reply:
We can merge after this is sorted